internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1012,9 +1074,12 @@ func (r *ClusterDeploymentsReconciler) createNewCluster(
 	}
 
 	if clusterInstall.Spec.IgnitionEndpoint != nil {
-		clusterParams.IgnitionEndpoint = &models.IgnitionEndpoint{
-			URL:           swag.String(clusterInstall.Spec.IgnitionEndpoint.Url),
-			CaCertificate: swag.String(clusterInstall.Spec.IgnitionEndpoint.CaCertificate),
+		var ignitionEndpoint *models.IgnitionEndpoint
+		ignitionEndpoint, err = r.parseIgnitionEndpoint(ctx, log, clusterInstall.Spec.IgnitionEndpoint)
+		if err == nil {
+			clusterParams.IgnitionEndpoint = ignitionEndpoint
+		} else {
+			log.WithError(err).Errorf("Failed to get and parse ignition ca certificate %s/%s", clusterInstall.Namespace, clusterInstall.Name)
```

Comment:
We cannot ignore errors, please return the error and handle it.
It should be reflected eventually in the conditions.

Reply:
Fixed